[Linux] - Unable to build AUR package on Arch Linux-based distos

> #### Checklist
> 
> <!-- You don't have to fill these out, but it would help us alot with managing open bug reports if you make sure to check these things first! -->
> 
> - [X] I am reporting exactly 1 bug with this issue.
> - [X] This bug hasn't already been reported.
> - [X] This bug hasn't already been fixed in the latest development build.

---
Compilation fails with: 

instrument/instruments_manager.cpp: In member function ‘void InstrumentsManager::addInstrument(int, InstrumentType, const string&)’:
instrument/instruments_manager.cpp:274:14: error: ‘invalid_argument’ is not a member of ‘std’
  274 |   throw std::invalid_argument("invalid instrument type");
      |              ^~~~~~~~~~~~~~~~
instrument/instruments_manager.cpp: In member function ‘void InstrumentsManager::addInstrument(AbstractInstrument*)’:
instrument/instruments_manager.cpp:339:14: error: ‘invalid_argument’ is not a member of ‘std’
  339 |   throw std::invalid_argument("invalid instrument type");
      |              ^~~~~~~~~~~~~~~~
instrument/instruments_manager.cpp: In member function ‘std::unique_ptr<AbstractInstrument> InstrumentsManager::removeInstrument(int)’:
instrument/instruments_manager.cpp:402:14: error: ‘invalid_argument’ is not a member of ‘std’
  402 |   throw std::invalid_argument("invalid instrument type");
      |              ^~~~~~~~~~~~~~~~
instrument/instruments_manager.cpp: In member function ‘void InstrumentsManager::cloneInstrument(int, int)’:
instrument/instruments_manager.cpp:476:14: error: ‘invalid_argument’ is not a member of ‘std’
  476 |   throw std::invalid_argument("invalid instrument type");
      |              ^~~~~~~~~~~~~~~~
instrument/instruments_manager.cpp: In member function ‘void InstrumentsManager::deepCloneInstrument(int, int)’:
instrument/instruments_manager.cpp:623:14: error: ‘invalid_argument’ is not a member of ‘std’
  623 |   throw std::invalid_argument("invalid instrument type");

@jpcima, can you please help with that?

--- a/BambooTracker/instrument/instruments_manager.cpp
+++ b/BambooTracker/instrument/instruments_manager.cpp
@@ -29,6 +29,7 @@
 #include <algorithm>
 #include <functional>
 #include <type_traits>
+#include <stdexcept>
 #include "instrument.hpp"
 #include "note.hpp"
 #include "utils.hpp"

